flyteidl/protos/flyteidl/core/execution.proto

```diff
@@ -73,6 +74,10 @@ message ExecutionError {
         SYSTEM = 2;
     }
     ErrorKind kind = 4;
+    // Timestamp of the error
+    google.protobuf.Timestamp timestamp = 5;
```

Comment:
I saw that you added this after @EngHabu asked why we don't add the timestamp here as well.
I don't mind adding the time stamp here in principle but why do we need it?

In the UI we don't really care about showing the timestamp I'd say, only the name of the worker.

Reply:
I thought it won't hurt to report the timestamp as well.

Reply:
Ok, sounds good 👍